modules/apps/analytics/analytics-client-js/package.json

```diff
@@ -9,7 +9,7 @@
 		"browser-tabs-lock": "^1.2.7",
 		"core-js": "^2.5.6",
 		"hash.js": "^1.1.7",
-		"object-hash": "^1.3.0",
+		"object-hash": "^2.0.1",
```

Comment:
@carolmariaabb, did you update this dependency here too, because you need to update to dependency to the whole portal, right?

Reply:
That’s right, @natocesarrego. I believe it's necessary because of the `yarn.lock` file.